autogpt_platform/backend/backend/blocks/google/_drive.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
+
+import base64
+from typing import Any, Literal
+
+from backend.data.model import GoogleDriveFile, GoogleDrivePickerField
+from backend.util.request import Requests
+from backend.util.type import MediaFileType
+
+DRIVE_API_URL = "https://www.googleapis.com/drive/v3/files"
+TRUSTED_ORIGINS = ["https://www.googleapis.com"]
```

Comment:
if we want to add trusted origin maybe we can simply create the request object as a global variable here too